tests/test_cgnsutilities.py

```diff
@@ -351,6 +353,40 @@ def test_mirror(self):
         self.assertEqual(15120 * 2, totalCells)
 
 
+class TestExtrude(unittest.TestCase):
+    def setUp(self):
+        x, y = np.meshgrid(np.linspace(0, 1, 20), np.linspace(0, 1, 13))
+        coords = np.zeros((x.shape[0], x.shape[1], 1, 3))
+        coords[:, :, 0, 0] = x
+        coords[:, :, 0, 1] = y
+
+        self.blk = Block("blk", coords.shape[:-1], coords)
+
+    def test_grid(self):
+        grid = Grid()
+
+        # Add the block and then add the same block but shifted by 1.0 in x
+        blk2 = Block("blk2", self.blk.coords.shape[:-1], self.blk.coords.copy())
+        blk2.coords[:, :, 0, 0] += 1.0
+        grid.blocks.extend([self.blk, blk2])
```

Comment:
Thanks for updating the tests. One clarification though. Is there a reason not to call the `grid.addBlock()` function? Functionally its doing the same as this statement (but just for a single block), but it encapsulates what is done and what datastructure is used.